modules/rseqc/innerdistance/main.nf

```diff
@@ -11,7 +11,7 @@ process RSEQC_INNERDISTANCE {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
 
-    conda (params.enable_conda ? "bioconda::rseqc=3.0.1" : null)
+    conda (params.enable_conda ? "bioconda::rseqc=3.0.1 'conda-forge::r-base>=3.5'" : null)
```

Comment:
```suggestion
    conda (params.enable_conda ? "bioconda::rseqc=3.0.1 conda-forge::r-base>=3.5" : null)
```

Reply:
Thanks @c-mertes ! I realise the other RSeQC modules are unaffected by this dependency but be great to update them too for consistency.

Reply:
Thanks for reviewing. It is added to the other ones.

The single ticks are required as it is pasted to the command line as plain text and hence, the `>` will through an error. It works with a pure `=`. At least on my Centos 8 installation.

Reply:
Ah, I didn't know that 👍🏽